airlock/templates/file_browser/request/file.html

```diff
@@ -34,6 +34,29 @@
       </form>
     {% endif %}
 
+    {% if content_buttons.move_file_button %}
+      {% if content_buttons.move_file_button.disabled %}
+        {% #button type="button" disabled=True tooltip=content_buttons.move_file_button.tooltip id="update-file-modal-button" %}
+          Move file to new group
```

Comment:
Suggest "Change filegroup"? When I first looked at the screenshots, this suggested to me that it was only for creating a new group